README.md

```diff
@@ -482,6 +482,63 @@ export { handler as default } from '@/next-video';
+- `streamType` ("on-demand" | "live"): The stream type of the video. Default is "on-demand".
+- `customDomain` (string): Assigns a custom domain to be used for Mux Video.
+- `beaconCollectionDomain` (string): Assigns a custom domain to be used for Mux Data collection. NOTE: Must be set before playbackId to apply to Mux Data monitoring.
+- `envKey` (string): This is the environment key for Mux Data. If you use Mux video this is automatically
+  set for you. If you use a different provider you can set this to your own key.
+- `disableTracking` (boolean): Disables Mux data tracking of video playback.
+- `disableCookies` (boolean): Disables cookies used by Mux Data.
+- `preferPlayback` ("mse" | "native"): Specify if `<mux-video>` should try to use Media Source Extension or native playback (if available). If no value is provided, `<mux-video>` will choose based on what's deemed optimal for content and playback environment.
+- `maxResolution` ("720p" | "1080p" | "1440p" | "2160p"): Specify the maximum resolution you want delivered for this video.
+- `minResolution` ("480p" | "540p" | "720p" | "1080p" | "1440p" | "2160p"): Specify the minimum resolution you want delivered for this video.
+- `programStartTime` (number): Apply PDT-based [instant clips](https://docs.mux.com/guides/create-instant-clips) to the beginning of the media stream.
+- `programEndTime` (number): Apply PDT-based [instant clips](https://docs.mux.com/guides/create-instant-clips) to the end of the media stream.
+- `assetStartTime` (number): Apply media timeline-based [instant clips](https://docs.mux.com/guides/create-instant-clips) to the beginning of the media stream.
+- `assetEndTime` (number): Apply media timeline-based [instant clips](https://docs.mux.com/guides/create-instant-clips) to the end of the media stream.
+- `renditionOrder` (string): Change the order in which renditions are provided in the src playlist. Can impact initial segment loads. Currently only support "desc" for descending order.
+- `metadataVideoId` (string): This is an arbitrary ID sent to Mux Data that should map back to a record of this video in your database.
+- `metadataTitle` (string): This is an arbitrary title for your video that will be passed in as metadata into Mux Data. Adding a title will give you useful context in your Mux Data dashboard. (optional, but encouraged)
+- `metadataViewerUserId` (string): If you have a logged-in user, this should be an anonymized ID value that maps back to the user in your database that will be sent to Mux Data. Take care to not expose personal identifiable information like names, usernames or email addresses. (optional, but encouraged)
+- `metadata*` (string): This metadata* syntax can be used to pass any arbitrary Mux Data metadata fields.
+- `playbackToken` (string): The playback token for signing the `src` URL.
+- `thumbnailToken` (string): The token for signing the `poster` URL.
+- `storyboardToken` (string): The token for signing the storyboard URL.
```

Comment:
Do you think we should separate Mux-specific attributes somehow? Or order them with a comment above saying "everything below is specific to Mux"?  Or at least try and order these by commonality. Might make it feel a little less unwieldy?

Like there are a ton of things in here that might feel like you could use them with any video source ("oh wow, this player let's me choose max resolution!") so I wouldn't want folks to get that impression.

Reply:
agreed, will make it more clear